analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SillyBitwiseOperation.Fixed.vb

```diff
@@ -196,3 +196,26 @@ Public Class Repro_4399
     End Sub
 
 End Class
+
+Public Class LocalFields
```

Comment:
C# version had this
```
        protected override object FindConstant(SemanticModel semanticModel, SyntaxNode node) =>
            IsFieldOrProperty(semanticModel, node, out var symbol) && !IsInSystemNamespace(symbol)
                ? null
                : node.FindConstantValue(semanticModel);
```
while VB version had only
```
        protected override object FindConstant(SemanticModel semanticModel, SyntaxNode node) =>
            node.FindConstantValue(semanticModel);
```
I've moved it to the base class, because the logic makes sense for both languages and is based only on symbols. This UT modification is translated version from C# to cover it for both languages.